src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -20,7 +19,7 @@ namespace Dynamo.Wpf.Extensions
     public class ViewLoadedParams : ReadyParams
     {
         private readonly DynamoView dynamoView;
-        private readonly DynamoViewModel dynamoViewModel;
+        public readonly DynamoViewModel dynamoViewModel;
```

Comment:
definitely don't want to do this for real. We do not want to expose the viewModel to all api users I don't think.

Reply:
Yes for sure, I will either change it to internal or use some other way to access the model object

Reply:
☠️